src/main/java/org/jabref/gui/logging/GuiWriter.java

```diff
@@ -29,7 +29,7 @@ public Collection<LogEntryValue> getRequiredLogEntryValues() {
 
     @Override
     public void write(LogEntry logEntry) throws Exception {
-        DefaultTaskExecutor.runInJavaFXThread(() -> LogMessages.getInstance().add(logEntry));
```

Comment:
I think there was a reason that when a background thread writes something to the log

Reply:
I belive this could lead to concurrency issues, as the guiwriter puts something in the logging list of the log dialog, which lives in the JavaFX-Thread, but GuiLogger is created before the FXThread is started. I think we had an issue about this some time ago...

Reply:
It should never be the responsibility of the source to care on which thread things are posted. The UI should ensure that it converts things from background threads to the ui thread. For example, we have the `BindingsHelper.forUi` method that does this for observable list. I've now used it in the log view dialog.